lib/refraction.js

```diff
@@ -18,6 +22,14 @@ function refract(value) {
     return new NullElement();
   } else if (Array.isArray(value)) {
     return new ArrayElement(value.map(refract));
+  } else if (typeof value === 'object') {
+    var element = new ObjectElement();
+
+    for (var key in value) {
+      element.set(key, value[key]);
+    }
+
+    return element;
```

Comment:
There wasn't any unit tests for `refract` function. I've added a full set in https://github.com/refractproject/minim/pull/122/commits/19d06d29b6f909334f0ef8860f3f8e96b37d7917.